src/EnergyPlus/DataGlobals.hh

```diff
@@ -112,8 +112,8 @@ struct DataGlobal : BaseGlobalStruct
     int TimeStep = 0;                // Counter for time steps (fractional hours)
     Real64 TimeStepZone = 0.0;       // Zone time step in fractional hours
     bool WarmupFlag = false;         // True during the warmup portion of a simulation
-    int StdOutputRecordCount = 0;    // Count of Standard output records
-    int StdMeterRecordCount = 0;     // Count of Meter output records
+    Int64 StdOutputRecordCount = 0;  // Count of Standard output records
+    Int64 StdMeterRecordCount = 0;   // Count of Meter output records
```

Comment:
Change StdOutputRecordCount & StdMeterRecordCount to Int64 (std::int64_t)

This increases the limit from 2,147,483,648 to 9,223,372,036,854,775,808 records

Reply:
Hmmmm.  OK, I believe you.  I guess that should be big enough :laughing: